space/decartToPolar.glsl

```diff
@@ -0,0 +1,19 @@
+/*
+author: Ivan Dianov
+description: Decart to polar transformation.
+use: decartToPolar(<vec2> st)
+license: |
+  Copyright (c) 2021 Ivan Dianov.
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef FNC_DECARTTOPOLAR
+#define FNC_DECARTTOPOLAR
+vec2 decartToPolar(in vec2 decart) {
```

Comment:
curious about the "decart" term, I always have seen it as "cart" for [cartesian](https://en.wikipedia.org/wiki/Cartesian_coordinate_system)

Do you mind also swapping the "to" for a 2 just to keep it coherent with color spaces that use `a2b()` structure. I know is not gramatically correct, but the two helps with the readability